src/xo-app/home/pool-item.js

```diff
@@ -44,8 +44,17 @@ import styles from './index.css'
     }
   )
 
+  const getHostMetrics = createSelector(
+    getPoolHosts,
+    hosts => ({
+      count: size(hosts),
+      cpus: reduce(hosts, (total, host) => total + +host.cpus.cores, 0),
+      memory: reduce(hosts, (total, host) => total + +host.memory.size, 0)
+    })
```

Comment:
Certainly, a getHostMetrics function taking optional filters or predicates as parameters, for example ?